scheduling/job_scheduling.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+"""
+Author : Mohit Kumar
+Job Sequencing Problem implemented in python
+"""
+from collections import namedtuple
+
+
+class Scheduling:
+    def __init__(self, jobs):
+        self.jobs = jobs
+
+    def schedule(self, n, deadline):
+        self.sdl = deadline
+        self.J = [self.jobs[1]]
+        self.x = 2
+        while self.x < n:
+            self.K = self.J.copy()
+            self.K.append(self.jobs[self.x])
+            self.x = self.x + 1
+            if self.feasible(self.K, self.sdl):
+                self.J = self.K.copy()
+
+        return self.J
+
+    def feasible(self, K, fdl):
```

Comment:
Python type hints? doctests? Avoid single letter variable names especially on for function parameters. See CONTRIBUTING.md.

Why do we need a fiddle?   Why force the reader to guess at what fdl is?